tf/persistent/README.md

```diff
@@ -1,7 +1,118 @@
-# Persistent testing resources
+## Persistent testing resources
 
-This module is for creating resources that should be persistent across tests.
-Right now it manages:
+This module is for creating resources that should be persistent across tests. There are no automated tests for the terraform files in this directory. 
 
-- The GKE cluster for running test pods
-- The Cloud Build CI Triggers for each repo
+Most of the resources used by these tests are set up and torn down for each test run. There are however some persistent resources which are managed by terraform (under [tf/persistent](../persistent/)) in the default terraform workspace:
+
+- The GKE cluster ([e2etest-default](https://pantheon.corp.google.com/kubernetes/clusters/details/us-central1/e2etest-default/details?project=opentelemetry-ops-e2e)) for running test pods.
+- The Cloud Build CI Triggers for each repo ([pantheon link](https://pantheon.corp.google.com/cloud-build/triggers?project=opentelemetry-ops-e2e)).
+- The default service for the created GAE application ([pantheon link](https://pantheon.corp.google.com/appengine/services?serviceId=default&versionId=v1&project=opentelemetry-ops-e2e)).
+
+If you want to modify any of the persistent resources, update the terraform configs and [apply](#apply) them, rather than changing them directly in
+cloud console UI. If anything goes wrong, [apply](#apply) the terraform configs to get
+them back to the desired state.
+
+### Apply
+
+The test runner has a convenience command `apply-persistent` which executes
+`terraform apply` on the persistent resources. This command will show you
+changes it would make and then ask for your confirmation before executing. It
+should be idempotent.
+
+Before you apply the persistent resources, make sure that you have built the docker image so that the changes in the terraform scripts can be reflected - 
+
+```bash
+# Build docker image of the test runner with latest changes
+docker build . -t opentelemetry-operations-e2e-testing:local
+```
+
+You can run the apply-persistent command by copying the following in a shell script - 
+
+```bash
+#!/bin/sh
+opentelemetry_operations_e2e_testing(){
+  docker run \
+    -e PROJECT_ID=${PROJECT_ID} \
+    -e GOOGLE_APPLICATION_CREDENTIALS=${GOOGLE_APPLICATION_CREDENTIALS} \
+    -v "${GOOGLE_APPLICATION_CREDENTIALS}:${GOOGLE_APPLICATION_CREDENTIALS}:ro" \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -it \
+    --rm \
+    opentelemetry-operations-e2e-testing:local \
+    "$@"
+}
+opentelemetry_operations_e2e_testing apply-persistent
+```
+
+and then executing the script using - 
+
+```
+sh <your_script_name>
+```
```

Comment:
why not just paste the above into your shell instead of saving a script?

Reply:
Good point, will do that.